doc/api/util.md

```diff
@@ -126,6 +126,10 @@ util.format(1, 2, 3); // '1 2 3'
 
 ## util.inherits(constructor, superConstructor)
 
+_Note: usage of `util.inherits()` is discouraged. Please use ES6 `class` and
+`extends` to get language level inheritance support, including an unbroken
+inheritance chain._
```

Comment:
That's better but it leaves the reader with unanswered questions about how
they're incompatible. Hmm... Perhaps some example code following would help
illustrate it?
On May 3, 2016 5:59 AM, "Robert Jefe Lindstädt" <notifications@github.com>
wrote:

> In doc/api/util.md
> <https://github.com/nodejs/node/pull/6514#discussion_r61875810>:
>
> > @@ -126,6 +126,10 @@ util.format(1, 2, 3); // '1 2 3'
> >
> >  ## util.inherits(constructor, superConstructor)
> >
> > +_Note: usage of `util.inherits()` is discouraged. Please use ES6 `class` and
> > +`extends` to get language level inheritance support, including an unbroken
> > +inheritance chain._
>
> Amended this. Hope pointing to the difference in "semantics" is clear
> enough that "the result will not be compatible":
>
>
> *Note: usage of util.inherits() is discouraged. Please use the ES6 class
> and extends keywords to get language level inheritance support. Also note
> that the two styles are semantically incompatible.*
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6514/files/e97064dbb212bbb717e9903d8e835140ed783380#r61875810>
>